### PR TITLE
Feature: Added carousel from react-elastic-carousel

### PR DESCRIPTION
**Pull request by acdavitt**
@julia-thea @ChrisRPeterson @dylanreid7 
Used react-elastic-carousel: 
 --npm install --save react-elastic-carousel
--npm install --save styled-components
--Set items to show to 4 and to display empty slots
--Future styling needed: hide disabled arrows and remove pagination dot at bottom

### CODE REVIEW

**Review by julia-thea — COMMENTED** [review]
@acdavitt yay to being able to install carousel module! 🎉
<del>What's your intention for this PR? Is it just installing the carousel package?</del>
Disregard my question. I see the corresponding Trello card to setup Material UI carousel.
I noticed that for some reason products cards got messed up in this PR. 
But if you okay with it, then I'll merge this PR

File: client/components/RelatedProducts/Related/RelatedProductsList.jsx [hunk 1]
```diff
@@ -3,14 +3,20 @@ import RelatedProductCard from './RelatedProductCard.jsx';
...
 import GridList from '@material-ui/core/GridList';
 // import Carousel from 'react-material-ui-carousel'
+import Carousel from 'react-elastic-carousel';
 
+//CSS styling for later so disabled arrow does not show:
+// .rec.rec-arrow:disabled {
+//   visibility: hidden;
+// }
 
 const RelatedProductsList = (props) => {
```
Comment (ChrisRPeterson) [inline client/components/RelatedProducts/Related/RelatedProductsList.jsx:13]: Try destructuring your props like so ```const RelatedProductsList = ({ relatedWithNameCatPrice }) => {```

**Review by ChrisRPeterson — APPROVED** [review]
It's looking really good!

File: client/components/RelatedProducts/Related/RelatedProductsList.jsx (hunk 1 above)
Comment (acdavitt) [reply client/components/RelatedProducts/Related/RelatedProductsList.jsx]: @ChrisRPeterson 
done! Thanks for suggesting!

**Comment by acdavitt** [comment]
@julia-thea 
I noticed that as well with the products card being all smashed together when my screen was small, but it still looked ok when I viewed it on a full screen.  Did the product cards look ok for you on a full screen? Definitely something to look into - responsive design for different screen sizes.

**Comment by julia-thea** [comment]
> @julia-thea
> I noticed that as well with the products card being all smashed together when my screen was small, but it still looked ok when I viewed it on a full screen. Did the product cards look ok for you on a full screen? Definitely something to look into - responsive design for different screen sizes.

**Comment by julia-thea** [comment]
@acdavitt the cards in the carousel definitely look like they are not rendering properly even on the full screen for me, but perhaps there is an issue with my setup.
Do you want to keep investigating it as a part of this PR, or can I get a go ahead to merge it?

**Comment by acdavitt** [comment]
@julia-thea I think go ahead and merge since it looks OK on my end.  And if others are also noticing the same thing, I can create a bug fix ticket for it.

**Pull request merged**